.vitepress/theme/styles/enhanced-tables.css

```
@@ -0,0 +1,527 @@
+.VPDoc table thead th {
+  color: var(--table-header-text) !important;
+  font-weight: var(--table-header-font-weight) !important;
+  font-size: var(--table-header-font-size);
+  text-align: left;
+  padding: var(--table-cell-padding) !important;
+  letter-spacing: 0.025em !important;
+  text-transform: none !important;
+  position: relative;
+  white-space: nowrap;
+  border-bottom: 2px solid rgba(255, 255, 255, 0.2);
+  background: transparent !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-image



(lint/suspicious/noShorthandPropertyOverrides)